workshop/exercise-3/README.md

```diff
@@ -1,79 +1,64 @@
-## Setup: Install Istio and Knative on Your Cluster
+## Deploy our First Application to Knative using the Knative Client (kn)
+A relatively new project, the knative client aims to make interacting with Knative a seamless experience for developers. We'll try deploying an application using `kn`, then we'll deploy the same application using `kubectl` and `.yaml` files for additional control. The application for this lab is a simple node.js with express app which returns the first n numbers of the Fibonacci sequence. Once the app is deployed, you can use it by making a GET request to the `/` endpoint with a number as the parameter. 
 
-Knative is currently built on top of both Kubernetes and Istio.
-If you want to learn more about Kubernetes or Istio, you can check out the
-labs [Kube101](https://github.com/IBM/kube101/tree/master/workshop) and
-[Istio101](https://github.com/IBM/istio101/tree/master/workshop).
-When you install Knative on IKS, it will install Istio for you
-automatically.
+# Our First Knative Service
+Knative Serving supports deploying and serving of serverless applications. Those applications will automatically scale up, and then back down to zero. In this exercise, we'll use the Knative Serving component to deploy our first application from a container image hosted on dockerhub. 
 
-### Install Knative
+The Knative Service object automatically manages the whole lifecycle for your workload. A service represents your app on Knative. Services control the creation of other objects to ensure that your app has a URL and a new revision for each update of the service.
 
-1. To install Knative, first make sure you have the latest Kubernetes
-   plugin:
+![](https://github.com/knative/serving/raw/master/docs/spec/images/object_model.png)
+
+# Deploy our Application to Knative using kn
+We've already created an image on dockerhub that contains the first version of our Fibonacci application. If we call the `/` endpoint, and pass in a `number` parameter, we should get the first `n` numbers of the Fibonacci sequence.
+
+1. Deploy the application:
 
     ```
-    ibmcloud plugin update
+    kn service create --image docker.io/ibmcom/fib-knative fib-knative
```

Comment:
just to save typing for the students, later we can remove the "docker.io/"  part

Reply:
is that not required?

Reply:
Nope - funny that people seem to be ok with the default repo being dockerhub. Will be interesting to see what happens if dockerhub were to ever vanish :-)